### ISSUE TITLE
array-bracker-spacing false positive

### ISSUE THREAD

**Issue by jlowcs** [buzinas/tslint-eslint-rules#162]
With the following rule:
```
"array-bracket-spacing": ["error", "never"]
```

And the following code:
```
const foo = [42/* , 51 */];
```

I get the following error:
```
array-bracket-spacing  There should be no space before "]"
```

Removing the `/* , 51 */` comment removes the error.
